android/app/src/main/java/com/votlin/android/ui/fragment/TalksListFragment.kt

```diff
@@ -41,56 +41,52 @@ class TalksListFragment : RootFragment<TalksListView>(), TalksListView {
     override val fragmentModule: Kodein.Module = Kodein.Module {
         bind<TalksListPresenter>() with provider {
             TalksListPresenter(
-                    view = this@TalksListFragment,
-                    errorHandler = AndroidErrorHandler(),
-                    executor = instance(),
-                    repository = instance())
+                view = this@TalksListFragment,
+                errorHandler = AndroidErrorHandler(),
```

Comment:
This is an error from our side, but error handler must be injected

Reply:
sure